api/app/tests/Behat/features-v2/report-submission/attaching-further-documents.feature

```diff
@@ -21,17 +21,17 @@ Feature: Attaching Further Documents
         Then I should be on the Lay homepage
         And a flash message should be displayed to the user confirming the document upload
         When I visit the documents step 2 page
-        Then I should see "testimage.png" listed as a previously submitted document
+        Then I should see "test_image.png" listed as a previously submitted document
```

Comment:
It would be good to have a test which covers the specific bug in the ticket, where a file with a hyphen in its name can be uploaded multiple times.